checkOpenPMD_h5.py

```diff
@@ -808,6 +811,12 @@ def check_particles(f, iteration, v, pic) :
     # root attributes at "/"
     result_array = np.array([0, 0])
     result_array += check_root_attr(f, verbose, extension_pic)
```

Comment:
5 lines above: remove the command line argument, switch to auto-detect only and print found extensions